.github/workflows/build-and-test-qemu.yml

```diff
@@ -68,4 +68,4 @@ jobs:
         with:
           name: meson-test-logs-${{ matrix.target }}
           path: |
-             ${{ github.workspace }}/build/meson-logs/testlog.txt
\ No newline at end of file
+             ${{ github.workspace }}/build/meson-logs/testlog.txt
```

Comment:
Was this just a missing trailing newline?

Reply:
Yeah, looks like I added the trailing newline while I was working on this 🤷